test/type_construction_test.rb

```diff
@@ -10081,4 +10081,34 @@ def test_block_splat_alias
       end
     end
   end
+
+  def test_type_narrowing_subclass
+    with_checker(<<-RBS) do |checker|
+class SuperClass
+end
+
+class SubClass < SuperClass
+end
+      RBS
```

Comment:
What do `SuperClass` and `SubClass` have to do with the rest of the test? It doesn't look like they are referenced or used.

Reply:
Thanks! 🙏